src/chttpd/src/chttpd.erl

```diff
@@ -873,6 +873,9 @@ error_info(conflict) ->
     {409, <<"conflict">>, <<"Document update conflict.">>};
 error_info({conflict, _}) ->
     {409, <<"conflict">>, <<"Document update conflict.">>};
+error_info({partition_overflow, DocId}) ->
+    Descr = <<"'", DocId/binary, "' exceeds partition limit">>,
```

Comment:
To be honest an error message that looks like `"'0000:0043' exceeds partition limit"` looks confusing to me, I think generic "Document update exceeds partition limit" looks more natural and similar to the rest of the error messages we are returning.

Reply:
What I was thinking is that for a single insert/update it would be obvious what doc id caused the overflow and on a  bulk update with a hundred of docs we'll get a hundred of the error messages iterating over all the keys like:
```
...
 {
    "id": "0000:0095",
    "error": "partition_overflow",
    "reason": "Partition limit exceeded due to update on '0000:0095'"
  },
  {
    "id": "0000:0096",
    "error": "partition_overflow",
    "reason": "Partition limit exceeded due to update on '0000:0096'"
  },
...
```
which seems repetitive.

But I agree that knowing the partition is useful and extracting the partition out of id just for the error message is a bit excessive, so let's go with your version.

Reply:
Fixed.